packages/schematics/migrations/update-6-0-0/update-6-0-0.ts

```diff
@@ -0,0 +1,633 @@
+import {
+  Rule,
+  SchematicContext,
+  Tree,
+  chain
+} from '@angular-devkit/schematics';
+import { stripIndents } from '@angular-devkit/core/src/utils/literals';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+
+import {
+  updateJsonInTree,
+  readJsonInTree,
+  getProjectConfig
+} from '../../src/utils/ast-utils';
+import { FormatFiles } from '../../src/utils/tasks';
+import { serializeJson } from '../../src/utils/fileutils';
+import { getSourceNodes } from '@schematics/angular/utility/ast-utils';
+import {
+  ngrxSchematicsVersion,
+  ngrxVersion,
+  angularVersion,
+  rxjsVersion
+} from '../../src/lib-versions';
+import { parseTarget } from '../../src/utils/cli-config-utils';
+import {
+  findFunctionCallExpressionStatement,
+  findRequireStatement,
+  findFunctionCalls,
+  findSpecDeclaration,
+  findTsNodeRegisterExpression
+} from './helpers';
+
+import * as ts from 'typescript';
+import * as path from 'path';
+import * as fs from 'fs';
+
+function updateKarmaConfigs(host: Tree, context: SchematicContext) {
```

Comment:
I don't think we can reuse the karma.conf.js. I would just replace it with the default karma.conf.js CLI generates for apps and libs